CHANGELOG.asciidoc

```diff
@@ -18,17 +18,12 @@ endif::[]
 ===== Bug fixes
 ////
 
-=== Unreleased
+[[release-notes-0.x]]
+=== Android agent version 0.x
 
-//${next_release_notes}
-
-[[release-notes-0.6.0]]
-==== 0.6.0 - //${release_date}
-
-[float]
-===== Features
```

Comment:
It looks like APM machine added this automatically. Is it possible to make this commented by default? It kinda clutters the release notes page otherwise.

Reply:
Yeah I automated the release notes not to forget to update them on a new release 😅 though I didn't know how it was going to affect the release notes page, though it's ok I think we can have it all commented until a new release is triggered, I'll take a look into it 👍